languages/go/tests/parity_test.go

```diff
@@ -145,12 +145,15 @@ func (MethodVariants) GetEmptyIter() interfaces.Iterator {
 	return IterableClass{}
 }
 
-// TODO: I don't think these make sense. Maybe as interfaces?
 type ParentClass struct{}
 
-type ChildClass struct{}
+// TODO: this kind of inheritance doesn't fit Go conventions
+// There is no way to use this kind of subclassing for composition, and
+// so it doesn't fit with our model of inheritance. Leaving for posterity
+// but these are not tested
+type ChildClass ParentClass
 
-type GrandchildClass struct{}
+type GrandchildClass ChildClass
```

Comment:
Why are we leaving this for posterity if there's no way to do this in Go?

Reply:
✂️ 'd out as no longer used in parity test suite.